app/frontend/src/pages/HomePage/HomePage.js

```diff
@@ -3,24 +3,34 @@ import Suggestion from './Suggestion';
 import Post from './Post';
 import Group from './Group';
 import Sidebar from '../../components/Sidebar'
+import kerimbahadir from "../../pages/Auth/images/kerimbahadir.jpg"
+import alikasap from "../../pages/Auth/images/alikasap.jpg"
+import zehrayildirim from "../../pages/Auth/images/zehrayildirim.jpg"
+import mahmutdemir from "../../pages/Auth/images/mahmutdemir.jpg"
+import fifa from "../../pages/Auth/images/fifa.jpg"
+import fortnite from "../../pages/Auth/images/fortnite.jpg"
+import pubg from "../../pages/Auth/images/pubg.jpg"
+import residentevil from "../../pages/Auth/images/residentevil.jpg"
+import rocketleague from "../../pages/Auth/images/rocketleague.jpg"
+import witcher from "../../pages/Auth/images/witcher.jpg"
```

Comment:
We can use an index.js file for these. Let's do something like this:

- Make a file inside the images folder as `index.js`
- Inside index.js, do smth like this:
 ```js
import kerimbahadir from "./kerimbahadir.jpg";
import alikasap from "./alikasap.jpg";
...

export {
    kerimbahadir,
    alikasap,
    ...
};
 ```
- Then, use the following:
```js
import * as images from "../../pages/Auth/images";

const groupData = [
    {
        image: images.pubg,
        ...
    }
    ...
]
```